tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/e2e-upgrade-test.go

```diff
@@ -25,7 +25,8 @@ var _ = Describe(
 	Label(tsparams.LabelEndToEndUpgrade), func() {
 
 		var (
-			clusterList []*clients.Settings
+			clusterList       []*clients.Settings
+			upgradeSuccessful = false
```

Comment:
suggest to initiate this as true (and set it to false when we enter the first test case), so that it is possible to run rollback test case on a upgraded cluster as a standalone test - could be helpful in failure analysis/rerun from specific stage.